.github/.renovaterc.json

```diff
@@ -16,6 +16,18 @@
       "updateTypes": ["minor", "patch"],
       "groupName": "all dependencies",
       "groupSlug": "all"
+    },
+    {
+      "updateTypes": ["major"],
+      "labels": ["changeset:patch"]
+    },
+    {
+      "updateTypes": ["minor"],
+      "labels": ["changeset:patch"]
```

Comment:
`changeset:minor`

Reply:
Yes. I am not super sure yet about it. Not every major dep bump to the lib is a major bump to consumers. 

Also I need to drop the usage of the action. It doesn't support mono repos.

Reply:
Ah ok, I thought it was a typo or something